### ISSUE TITLE
i3 bar icons malformed

### ISSUE THREAD

**Issue by PreciousChicken** [regolith-linux/regolith-i3blocks#1]
![i3_bar_icon_error](https://user-images.githubusercontent.com/13577413/59301700-22e02080-8c8a-11e9-9e76-62c1be493847.png)
### Summary
The i3 bar icons are not displaying correctly

### Steps to reproduce
Regolith installed as PPA on Ubuntu 18.04 LTS.  Malformed icons apparent on login.

### Expected results
There should be a cog symbol to represent the current processor load and a calendar to represent the time/date.

### Actual results
As shown in screenshot - malformed icons.

### Comments
I thought the original problem was that I did not have the Source Code Pro Medium font installed; I did this, removed the config files and reinstalled Regolith.  This failed to work.   I then copied using a USB the i3blocks config files from a laptop which I'd used the Live CD on, where the icons were displaying correctly; however on viewing them on the affected machine they were again misconfigured.  

I appreciate this is a more of a general Linux/Ubuntu issue than an Regolith one, however other people may be suffering from this.  I eventually mitigated the problem by simply finding other icons (though not the originals, which I can't seem to find) and copy/pasting them into the config file, but I thought the issue should be raised.

**Comment by kgilmer** [comment]
Thank you @PaleAugust for the perfectly detailed report.  This seems like a regression.  I believe the missing font is called font-awesome (https://launchpad.net/ubuntu/bionic/+package/fonts-font-awesome).  You may try installing it manually or wait until I fix the bug and just take the update from the PPA.  Either way your system will end up in the same state, but if you do end up installing the font and it still does not work, please let me know.

**Comment by kgilmer** [comment]
I installed a fresh copy of Ubuntu 18.04 on a laptop and then installed Regolith from the PPA.  I wasn't able to reproduce your issue so I'm guessing that something may have gotten mangled on your machine at some point.  In any case, if after installing fonts-font-awesome and/or a fresh install of Regolith from the PPA, if you still have the issue please reopen!

Thanks again for the detailed report, very helpful.

**Comment by PreciousChicken** [comment]
![regolith_icon_fix_2](https://user-images.githubusercontent.com/13577413/59355932-d1827080-8d1f-11e9-8a99-126b2b69f0fb.png)

TL;DR: Suggest you change line 230 in file `.config/i3-regolith/config` to read `font pango:Source Code Pro Medium 13, FontAwesome 13`?

@kgilmer Thanks very much for the super quick response to this.

You were right, this wasn't a clean install of Ubuntu; it was one I'd been using for a while, so something has obviously gone wrong with my setup.

However now that I knew the name of the font you were using I was able to check the most obvious - was the font installed?  It was, I could see the font in GNOME Character Map - but for some reason it was mangled, when I tried to reference the characters in the terminal or copy them from the character map I still got the malformed icons.

Some googling later, I found that this [exact problem had happened to previous i3 users](https://www.reddit.com/r/i3wm/comments/6m75uf/font_awesome_characters_have_become_corrupt/).  The suggestion here was to edit `.config/i3-regolith/config` changing the line:

`font pango:Source Code Pro Medium 13`

to 

`font pango:Source Code Pro Medium 13, FontAwesome 13` 

I saved the change, logged out, logged in again and icons are correct - see screenshot.  They continue to appear malformed when viewing in neovim and elsewhere, which is strange; but that's a problem for another day...

You might want to consider changing the config file?  Making the change will not affect those with a clean install, but will stop a problem starting for others in my situation?  My limited understanding is that by specifying two fonts you instruct the program if it can't find a character in the first font set, to look in the second; so if the cog/date character is not in Source Code Pro they will be sourced from FontAwesome?  I'm guessing your install figured this out without the hint - mine for whatever reason didn't.  I could have this wrong though - you will undoubtedly have a deeper understanding of this than me.  I also notice searching the repo that FontAwesome used to be specifically referenced, but you have since deleted; so maybe you have a reason...

Either way, I've solved my problem (thanks to your help); and hopefully illustrated what might be an issue for others.  Very much appreciate your assistance (and this great distro).

NB - If you wanted I could put in a pull request to add it, however actually never contributed to an open source project before... so would be a big (and cool) step!  Let me know.

**Comment by kgilmer** [comment]
@PaleAugust , fantastic analysis!  I do think you found a bug and the Regolith project would be honored to have your first open source contribution.  Actually, this detailed bug report is a very valuable open source contribution.  In any case, I'll await your pull request!  Please let me know if you get stuck or have any questions. :+1:
